models/models.go

```diff
@@ -105,9 +105,11 @@ type StepModel struct {
 	//  steps will run which are marked with IsAlwaysRun.
 	IsSkippable *bool `json:"is_skippable,omitempty" yaml:"is_skippable,omitempty"`
 	// RunIf : only run the step if the template example evaluates to true
-	RunIf   *string                `json:"run_if,omitempty" yaml:"run_if,omitempty"`
-	Timeout *int                   `json:"timeout,omitempty" yaml:"timeout,omitempty"`
-	Meta    map[string]interface{} `json:"meta,omitempty" yaml:"meta,omitempty"`
+	RunIf   *string `json:"run_if,omitempty" yaml:"run_if,omitempty"`
+	Timeout *int    `json:"timeout,omitempty" yaml:"timeout,omitempty"`
+	// The timeout (in seconds) until a Step with no output (stdout/stderr) is aborted
+	NoOutputTimeout *int64                 `json:"no_output_timeout,omitempty" yaml:"no_output_timeout,omitempty"`
```

Comment:
As far as I see, the default value here is `-1`, so this property will be included by default / with the default value.

Reply:
Changed default to 0. -1 is accepted still, meaning timeout is disabled.